docker/ubuntu-gtsam-python/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Get the base Ubuntu/GTSAM image from Docker Hub
+FROM dellaert/ubuntu-gtsam:bionic
+
+# Install pip
+RUN apt-get install -y python-pip python-dev
+
+# Install python wrapper requirements
+RUN pip install -r /usr/src/gtsam/cython/requirements.txt
```

Comment:
We need to check if `pip` invokes the built in `pip2` (which is deprecated) or `pip3`.

Reply:
what about using `python3 -m pip install ...` to be 100% sure?

Reply:
That would be the best option. :slightly_smiling_face: